### PR TITLE
Bite-sized adjustment for flake8, more f-string conversions

### PR DESCRIPTION
**Pull request by pgunn**
Made my local flake8 (3.7.8) happy, also did a few f-string conversions on the way.

### CODE REVIEW

File: django/applications/catmaid/apps.py [hunk 1]
```diff
@@ -311,11 +310,11 @@ def validate_configuration(self) -> None:
             if not isinstance(getattr(settings, field), allowed_types): # type: ignore
                 current_type = type(getattr(settings, field))
                 if len(allowed_types) == 1:
-                    raise ImproperlyConfigured("Please make sure settings field %s "
-                            "is of type %s (current type: %s)" % (field, data_type, allowed_types[0]))
```
Comment (pgunn) [inline django/applications/catmaid/apps.py:315]: I'm pretty sure this was a bug (last parameter was wrong)

**Comment by pgunn** [comment]
@clbarnes Hey, what do you think?

**Comment by clbarnes** [comment]
Sorry I haven't had a chance to look at this, I've just moved back to the UK so I'm a bit all over the place. Away over the weekend but I'll get to it early next week, I expect.

File: django/applications/catmaid/apps.py (hunk 1 above)
Comment (clbarnes) [reply django/applications/catmaid/apps.py:315]: Agreed.

**Pull request merged**